sqldelight-runtime/src/commonMain/kotlin/com/squareup/sqldelight/logs/LogSqliteDriver.kt

```diff
@@ -0,0 +1,80 @@
+package com.squareup.sqldelight.logs
+
+import com.squareup.sqldelight.Transacter
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.db.SqlPreparedStatement
+
+class LogSqliteDriver(
+        private val sqlDriver: SqlDriver,
```

Comment:
File should be two-space indent. These should only be one level indented, not two.